pages/api/export.tsx

```diff
@@ -0,0 +1,91 @@
+import { NextApiRequest, NextApiResponse } from "next";
+
+import * as fs from "fs";
+
+import { IFormatedEvent } from "..";
+
+import { createEvents, DateArray, EventAttributes } from "ics";
+
+// Convert events to ICS format
+function convertEventsToICS(events: IFormatedEvent[]) {
+  const icsEvents: EventAttributes[] = events.map((event: IFormatedEvent) => {
+    const s: DateArray = [
+      event.start.getFullYear(),
+      event.start.getMonth() + 1,
+      event.start.getDate(),
+      event.start.getHours(),
+      event.start.getMinutes(),
+    ];
+    const e: DateArray = [
+      event.end.getFullYear(),
+      event.end.getMonth() + 1,
+      event.end.getDate(),
+      event.end.getHours(),
+      event.end.getMinutes(),
+    ];
```

Comment:
```suggestion
    function buildDate(date: Date): DateArray {
      return ...
    }
```
Create a function like so (outside of this one).